website/src/components/posts/CollegeAdmissions/essay/EssayContainer.js

```diff
@@ -12,15 +12,16 @@ import "./essay.css";
 const INITIAL_WORD_LENGTH = 3;
 const wordSections = [
   {
-    plain: "This is our example",
+    plain: "This is our example ",
```

Comment:
why was there space added here but not on the other `plain` attribute strings? or is this extra space just a typo?

Reply:
after going to the deployment i see that it is just a typo --- we an fix this later when we add the actual sample essay in, so no need to worry about this now